src/pages/ValidateLoginPage/index.website.tsx

```diff
@@ -91,4 +95,9 @@ export default withOnyx<ValidateLoginPageProps<ValidateLoginPageOnyxProps>, Vali
     account: {key: ONYXKEYS.ACCOUNT},
     credentials: {key: ONYXKEYS.CREDENTIALS},
     session: {key: ONYXKEYS.SESSION},
+    autoAuthState: {
+        key: ONYXKEYS.SESSION,
+        selector: (session) => session?.autoAuthState,
+        initWithStoredValues: false,
+    },
```

Comment:
Instead of clearing the autoAuthState on mounted, I connect a new onyx key specifically fo fetch the autoAuthState with initWithStoredValues as false so the value is always undefined at first render.